source/patterns/@aws-solutions-constructs/aws-alb-lambda/README.md

```diff
@@ -24,43 +24,65 @@
 
 This AWS Solutions Construct implements an an Application Load Balancer to an AWS Lambda function
 
-Here is a minimal deployable pattern definition in Typescript:
+Here is a minimal deployable pattern definition:
 
+Typescript
 ``` typescript
-
-  // Obtain a pre-existing certificate from your account
-  const certificate = acm.Certificate.fromCertificateArn(
-        scope,
-        'existing-cert',
-        "arn:aws:acm:us-east-1:123456789012:certificate/11112222-3333-1234-1234-123456789012"
-      );
-  const props: AlbToLambdaProps = {
+import { AlbToLambda, AlbToLambdaProps } from '@aws-solutions-constructs/aws-alb-lambda';
+import * as acm from 'aws-cdk-lib/aws-certificatemager';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+// Obtain a pre-existing certificate from your account
+const certificate = acm.Certificate.fromCertificateArn(
+    this,
+    'existing-cert',
+    "arn:aws:acm:us-east-1:123456789012:certificate/11112222-3333-1234-1234-123456789012"
+);
+const props: AlbToLambdaProps = {
     lambdaFunctionProps: {
-      code: lambda.Code.fromAsset(`${__dirname}/lambda`),
-      runtime: lambda.Runtime.NODEJS_14_X,
-      handler: 'index.handler'
+        code: lambda.Code.fromAsset(`${__dirname}/lambda`),
```

Comment:
These attributes should be indented 2. The only 4 space indent is multiple arguments for a function call.